src/main/resources/worker.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# Downloads and runs an analyst worker.
+# This will be run through Java MessageFormat, and the following items will be available by wrapping
+# the relevant index in curly braces
+# 0: the URL to grab the worker JAR from
+# 1: the AWS log group to use
+# 2: the worker configuration to use
+
+# prep the system: install log agent, java
+yum -y install awslogs java-1.8.0-openjdk
+
+# first things first: set up logging
+LOGFILE=/var/log/analyst-worker.log
+
+echo Starting analyst worker at `date` > $LOGFILE
+
+# make it so that the worker can write to the logfile
+chown ec2-user:ec2-user $LOGFILE
+chmod 664 $LOGFILE # Log agent needs to read log file
+
+cat > /etc/awslogs/awslogs.conf <<EOF
```

Comment:
It would be good to have a short comment mentioning "heredocs" to save someone in the future confusion when they've never seen this before.